### PR TITLE
Added Department in category for collection (dataverse)

### PR DESCRIPTION
**Pull request by rbhatta99**
## Related Issues

- connects to #2829 

## Pull Request Checklist

- [ ] Unit [tests][] completed
- [ ] Integration [tests][]: None
- [ ] Deployment requirements, [SQL updates][], [Solr updates][], etc.: None
- [ ] [Documentation][docs] completed
- [ ] Merged latest from "develop" [branch][] and resolved conflicts

[tests]: http://guides.dataverse.org/en/latest/developers/testing.html
[SQL updates]: https://github.com/IQSS/dataverse/tree/develop/scripts/database/upgrades
[Solr updates]: https://github.com/IQSS/dataverse/blob/develop/conf/solr/4.6.0/schema.xml
[docs]: http://guides.dataverse.org/en/latest/developers/documentation.html
[branch]: http://guides.dataverse.org/en/latest/developers/branching-strategy.html

### CODE REVIEW

File: src/main/java/edu/harvard/iq/dataverse/Dataverse.java [hunk 1]
```diff
@@ -105,7 +106,7 @@ public void setDataverseType(DataverseType dataverseType) {
     private final String uncategorizedString = "Uncategorized";
 
     /**
-     * @todo Don't hard code these as English.
+     * @todo Don't hard code these as English. Use bundleUtils
```
Comment (mheppler) [inline src/main/java/edu/harvard/iq/dataverse/Dataverse.java]: @rbhatta99 It looks like you can remove this todo if you add bundle references to the "Laboratory" and "Research Group" text in this same function you reference the bundle for "Department" in line 128.

**Comment by mheppler** [comment]
@rbhatta99 I added a comment to your pull request in regards to a todo in Dataverse.java. @scolapasta might also be able to confirm this improvement as well.

File: src/main/java/edu/harvard/iq/dataverse/Dataverse.java (hunk 1 above)
Comment (rbhatta99) [inline src/main/java/edu/harvard/iq/dataverse/Dataverse.java]: I can do that. I can actually change all of them if need be, as they all are given in the bundle. Should I do it in this branch only?

Comment (mheppler) [inline src/main/java/edu/harvard/iq/dataverse/Dataverse.java]: Yeah, all of them would be great. Sorry, GitHub only displayed a snippet of truncated code, so I only referenced those two.

Comment (rbhatta99) [inline src/main/java/edu/harvard/iq/dataverse/Dataverse.java]: Sure, il do that!

**Comment by rbhatta99** [comment]
@mheppler I completed the todo in dataverse.java
@jggautier reorganised the categories list alphabetically

**Review by mheppler — APPROVED** [review]
Looks good!

**Pull request merged**